### DISCUSSION TITLE
Support for headless mode

### DISCUSSION THREAD

**Discussion by nattb8** [Voltstro-Studios/UnityWebBrowser#370] [Requests]
Is it possible to introduce a headless mode for UnityWebBrowser to allow it to function without rendering a UI or requiring interaction? This feature would enable background processing, such as executing scripts or fetching data, without visual or interactive components.

**Comment by Voltstro** [comment]
Doesn't look too hard to implement, it would effectively just be the `WebBrowserClient` but without the `pixelDataLoopThread` running.

I'll take a proper look over the weekend.

**Comment by nattb8** [comment]
Hi again @Voltstro, any updates on this support?

**Comment by Voltstro** [comment]
Feature has been added in release 2.2.6.

Some docs here: https://projects.voltstro.dev/UnityWebBrowser/latest/articles/user/runtime/#headless-mode

**Reply by nattb8** [reply]
@Voltstro Thank you!
